src/FluentValidation.Extensions/FluentValidation.Extensions.AutoMapper.csproj

```diff
@@ -34,12 +35,31 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="8.1.2" />
     <PackageReference Include="JetBrains.Annotations" Version="2018.2.1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="FluentValidation" Version="11.2.0" />
+  </ItemGroup>
+	
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461'">
+    <PackageReference Include="FluentValidation" Version="9.5.4" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45'">
+	<PackageReference Include="FluentValidation" Version="8.6.3" />
+  </ItemGroup>
+	
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="AutoMapper" Version="8.0.0" />
```

Comment:
I would expect that this would be version 11 for automapper here?

Or is version 8 still fine here?

In that case the predef `AUTOMAPPER11` seems incorrect?

Reply:
Yes, this was a mistake. It have changed it to the latest version of 11. Now I'm struggling because [`Mapper.GetAllTypeMaps()` was removed](https://github.com/AutoMapper/AutoMapper/issues/1252). I am trying to gethis working in my [*automapper-11-broken*  branch](https://github.com/goldsam/FluentValidation.Extensions.AutoMapper/blob/4f475b8e8304f614c3aeda1b38a50111204b6375/src/FluentValidation.Extensions/FluentValidationPropertyNameResolver.cs#L33).

Reply:
I'll get your branch and see if I understand it...

Reply:
@goldsam 

I fixed it by accessing internal members using reflection.
See my branch
https://github.com/StefH/FluentValidation.Extensions.AutoMapper/pull/3


(the .net6.0 example works fine, the older example is broken i think... I need to check that,.,,)